rostsd_gen/index.js

```diff
@@ -64,6 +64,7 @@ function getPkgInfos(rootDir) {
 
     for (let filename of files) {
       const typeClass = fileName2Typeclass(filename);
+      if (typeClass.type !== 'srv' && typeClass.type !== 'action' && typeClass.type !== 'msg') continue;
```

Comment:
Hi @locmark thanks for submitting the PR, my first thought is this way does hide the exception, but not solve it. I'm going to take a look later, thanks!

Reply:
Yeah, it doesn't solve it at all. But if solving it neatly is going to take like forever, I consider it better than not working module. 
Just to be clear, I'm not implying that it should be merged, but maybe some temporary solution should be made.